agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/HttpClientTracer.java

```diff
@@ -172,7 +171,7 @@ private Span onResponse(final Span span, final RESPONSE response) {
     return span;
   }
 
-  private String spanNameForRequest(final REQUEST request) {
+  protected String spanNameForRequest(final REQUEST request) {
```

Comment:
i think this can stay private for now at least

Reply:
It is needed in the branch 3.